apix-files/README.md

```diff
@@ -0,0 +1,40 @@
+# API Explorer file server
+
+API Explorer (aka APIX) can be uses some JSON-formatted "index" files to augment the information provided in a specification.
+
+To use this server, clone this repository and:
+
+```sh
+yarn && yarn build
+yarn mine
+cd apix-files
+yarn serve
+```
+
+This will start the API Explorer file server at `http://localhost:30000`
+
+## Mining the mother lode
```

Comment:
Any issues with mother lode as a politically correct term?

Reply:
I don't *think" that's a charged term. It's taken from mining but I guess we can change it to "examples index" and "examplesIndex.json" to be less obscure